### PR TITLE
Middleware 1.3.42

### PR DESCRIPTION
**Pull request by mijomilicevic**
- [x] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

**Review by Bluette-C-Riviere — APPROVED** [review]

File: middleware/2023-01-19-1.3.42.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+authors: poscreator
+slug: middleware/1.3.42
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.42 (Germany)
+In this version of the Middleware we reworked the way we build the reference trees in our DSFinV-K export. Additionally, we introduced some changes to the handling of specific business cases in our DSFinV-K Export.
```
Comment (TSchmiedlechner) [inline middleware/2023-01-19-1.3.42.md:8]:
```suggestion
In this version of the Middleware we reworked the way we build the reference trees in our DSFinV-K export to increase its performance in some specific cases. Additionally, we introduced some changes to the handling of specific business cases in our DSFinV-K export, and fixed a format issue that affected fiskaly TSEs and AmadeusVerify.
```

File: middleware/2023-01-19-1.3.42.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.42_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.42_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.42
+
+## Bug Fix: fiskaly certificate displayed in wrong format
+We fixed a bug where in some cases the fiskaly certificate was not shown correctly in the DSFinV-K Export.
+
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.42_
```
Comment (TSchmiedlechner) [inline middleware/2023-01-19-1.3.42.md:39]: Should be the fiskaly package only, right?

File: middleware/2023-01-19-1.3.42.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.42_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.42
+
+## Feature: Target Framework Update for Android App
+We´ve updated the target Framework of our Android App to 12.0+ to be compliant with the Google Target API Level Policy.
+
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.42_
```
Comment (TSchmiedlechner) [inline middleware/2023-01-19-1.3.42.md:47]: Should be the AndroidLaunchers only, right? (HTTP and gRPC).

File: middleware/2023-01-19-1.3.42.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+## Bug Fix: fiskaly certificate displayed in wrong format
+We fixed a bug where in some cases the fiskaly certificate was not shown correctly in the DSFinV-K Export.
+
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.42_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.42_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.42
+
+## Feature: Target Framework Update for Android App
+We´ve updated the target Framework of our Android App to 12.0+ to be compliant with the Google Target API Level Policy.
```
Comment (TSchmiedlechner) [inline middleware/2023-01-19-1.3.42.md:44]:
```suggestion
We´ve updated the target Framework of our Android App to 12.0+ to both support this Android version and be compliant with the Google Target API Level Policy.
```

**Comment by TSchmiedlechner** [comment]
I think the changes in the TOC and the readme.md are also missing :)

**Pull request merged**